src/components/networked.js

```diff
@@ -373,39 +381,42 @@ AFRAME.registerComponent('networked', {
   },
 
   updateComponent: function (el, componentName, data) {
-    if(!NAF.options.useLerp) {
+    const transformComponents = ["position", "rotation", "scale"]
+    if(!NAF.options.useLerp || !transformComponents.includes(componentName)) {
       el.setAttribute(componentName, data);
       return;
     }
 
-    var buffer = null;
-    var interpolationBuffer = this.interpolationBuffers.find((item) => item.el === el);
-    if (!interpolationBuffer) {
-      buffer = new InterpolationBuffer(InterpolationBuffer.MODE_LERP, 0.1);
-      this.interpolationBuffers.push({ buffer: buffer, el: el });
-    } else {
-      buffer = this.interpolationBuffers.find((item) => item.el === el).buffer;
+    var bufferInfo = this.bufferInfos.find((info) => info.object3D === el.object3D);
+    if (!bufferInfo) {
+      bufferInfo = { buffer: new InterpolationBuffer(InterpolationBuffer.MODE_LERP, 0.1),
+                     object3D: el.object3D,
+                     componentNames: [componentName] };
+      this.bufferInfos.push(bufferInfo);
+    }
+    var buffer = bufferInfo.buffer;
+    var componentNames = bufferInfo.componentNames;
+    if (!componentNames.includes(componentName)){
+      componentNames.push(componentName);
     }
 
     switch(componentName) {
       case "position":
         buffer.setPosition(this.bufferPosition.set(data.x, data.y, data.z));
-        break;
+        return;
       case "rotation":
         this.conversionEuler.set(DEG2RAD * data.x, DEG2RAD * data.y, DEG2RAD * data.z);
         buffer.setQuaternion(this.bufferQuaternion.setFromEuler(this.conversionEuler));
-        break;
+        return;
       case "scale":
         buffer.setScale(this.bufferScale.set(data.x, data.y, data.z));
-        break;
-      default:
-        el.setAttribute(componentName, data);
-        break;
+        return;
     }
+    NAF.log.error("Could not set value in interpolation buffer.", el, componentName, data, bufferInfo);
```

Comment:
This line is impossible to reach currently, but we end on a switch statement and I thought maybe if code above this line changes, this line might become reachable by accident, which would be an error.